modules/unsupported/css/src/main/java/org/geotools/styling/css/CssTranslator.java

```diff
@@ -219,6 +215,27 @@
         }
     };
 
+    @SuppressWarnings("serial")
+    static final Map<String, String> CONTRASTENHANCMENT_VENDOR_OPTIONS = new HashMap<String, String>() {
+        {
+            put("-gt-raster-contrast-enhancement-algorithm", "algorithm");
+
+            put("-gt-raster-contrast-enhancement-minvalue", "minValue");
+            put("-gt-raster-contrast-enhancement-maxvalue", "maxValue");
```

Comment:
Jody, is this kind of feedback really valuable other than having spending twice the time to deliver this work?